### DISCUSSION TITLE
RX 6700 (non XT version) won't be controlled

### DISCUSSION THREAD

**Discussion by ninja-gaijin** [Rem0o/FanControl.Releases#1390] [Q&A]
So Fan Control detects the GPU, but doesn't control it, even if set on manual.
MSI Afterburner does work for it.

It's a Sapphire Pulse RX 6700 GPU (non XT version).

Is this GPU not supported or am I doing someting wrong?

edit: I saw on the FAQ:
Q: My new AMD card can't be controlled, why?
A: Newer AMD cards are not supported. If you can provide a working sample of code that works with these newer cards, please share!

**Comment by Rem0o** [comment]
^ As the FAQ said.
